### DISCUSSION TITLE
v3.1.1

### DISCUSSION THREAD

**Discussion by mehalter** [AstroNvim/AstroNvim#1631] [Announcements]
## What's Changed
* Fix for #1628 
* Update pinned plugin commits

**Full Changelog**: https://github.com/AstroNvim/AstroNvim/compare/v3.1.0...v3.1.1

<hr /><em>This discussion was created from the release <a href='https://github.com/AstroNvim/AstroNvim/releases/tag/v3.1.1'>v3.1.1</a>.</em>

**Comment by ScopeSV** [comment]
I hate the new key bindings. I'm also not working on a US keyword. [b to change panes makes no sense. 
I want shift + h / l back. It made perfect sense to use shift to move between panes and ctrl to move between windows.

Any easy way to turn this back on, without screwing up something?

**Reply by mehalter** [reply]
Yeah definitely! Check out the v3 migration guide: https://astronvim.com/Configuration/v3_migration

**Comment by ScopeSV** [comment]
Thanks, that worked. Quite the job moving over to v3.
I'm trying to get prettier to work. I found this config https://astronvim.com/Recipes/advanced_lsp where I added the content to user/plugins/core.lua 
It doesn't seem like its working. Seems like only the default formatter is doing the job.

Any good pointers on how to get it set up correctly? It was very easy in the pre v3 config, only had to comment out a line.
Kind of like this: https://github.com/AstroNvim/user_example/blob/main/plugins/null-ls.lua
I tried this as well, didnt work

**Reply by mehalter** [reply]
I would recommend joining the discord as there are a lot of people there who can you debug your condig
